### PR TITLE
update tsg_install

### PR DESCRIPTION
**Pull request by ranjithwingrider**
Updated 
Signed-off-by: ranjithwingrider ranjith.raveendran@openebe.io

### CODE REVIEW

File: docs/tsg_install.md [hunk 1]
```diff
@@ -136,7 +137,14 @@ kubectl delete crd volumesnapshotdatas.volumesnapshot.external-storage.k8s.io
 kubectl delete crd volumesnapshots.volumesnapshot.external-storage.k8s.io
 ```
  
-
+ ## On Centos 7.5,While trying openebs 0.7,getting below error in the node-disk-manager pod.
```
Comment (muratkars) [inline docs/tsg_install.md]:
```suggestion
 ## While running OpenEBS v0.7 on Centos 7.5 getting below error in the node-disk-manager pod.
```

Comment (mahebbar) [reply docs/tsg_install.md]: Centos 7.5, the following error occurs in the node-disk...

File: docs/tsg_install.md [hunk 2]
```diff
@@ -136,7 +137,14 @@ kubectl delete crd volumesnapshotdatas.volumesnapshot.external-storage.k8s.io
...
+ 
+ ```
+ container_linux.go:247: starting container process caused "process_linux.go:359: container init caused \"rootfs_linux.go:53: mounting \\\"/proc/1/mounts\\\" to rootfs \\\"/mnt/docker/devicemapper/mnt/c6102aa3daf7ff291b1595fd6a9bf1367fd6d013498cc0a36442d9de4b968f3d/rootfs\\\" at \\\"/mnt/docker/devicemapper/mnt/c6102aa3daf7ff291b1595fd6a9bf1367fd6d013498cc0a36442d9de4b968f3d/rootfs/host/mounts\\\" caused \\\"permission denied\\\"\""
+ ```
+ ### Troubleshooting
+ 
+The hosts mount file is being attached to the ndm pod is to detect if the discovered disk is an OS disk. If selinux is enabled, this mount file cannot be able to attach by ndm. Try to disable selinux by  `setenforce 0` , which will allow to detect the disks.
```
Comment (mahebbar) [inline docs/tsg_install.md]: The host's ??
mount file that is attached to the ndm...
...ndm cannot attach this mount file
Disable selinux using `setenforce 0` command, which will allow detecting the disks.

File: docs/tsg_install.md [hunk 3]
```diff
@@ -136,7 +137,14 @@ kubectl delete crd volumesnapshotdatas.volumesnapshot.external-storage.k8s.io
 kubectl delete crd volumesnapshots.volumesnapshot.external-storage.k8s.io
 ```
  
-
+ ## While running OpenEBS v0.7 on Centos 7.5 getting below error in the node-disk-manager pod.
```
Comment (mahebbar) [inline docs/tsg_install.md]: Running OpenEBS v0.7 displays an error in the node-disk-manager pod

File: docs/tsg_install.md [hunk 4]
```diff
@@ -136,7 +137,14 @@ kubectl delete crd volumesnapshotdatas.volumesnapshot.external-storage.k8s.io
...
+ 
+ ```
+ container_linux.go:247: starting container process caused "process_linux.go:359: container init caused \"rootfs_linux.go:53: mounting \\\"/proc/1/mounts\\\" to rootfs \\\"/mnt/docker/devicemapper/mnt/c6102aa3daf7ff291b1595fd6a9bf1367fd6d013498cc0a36442d9de4b968f3d/rootfs\\\" at \\\"/mnt/docker/devicemapper/mnt/c6102aa3daf7ff291b1595fd6a9bf1367fd6d013498cc0a36442d9de4b968f3d/rootfs/host/mounts\\\" caused \\\"permission denied\\\"\""
+ ```
+ ### Troubleshooting
+ 
+The hosts mount file is being attached to the ndm pod is to detect if the discovered disk is an OS disk. If selinux is enabled, this mount file cannot be able to attach by ndm. Try to disable selinux by  `setenforce 0` , which will allow to detect the disks.
```
Comment (mahebbar) [inline docs/tsg_install.md]: The hosts mount file that is attached to the ndm pod is to detect...
this mount file cannot be able to attach by ndm. = ndm will be unable to attach the mount file.

Comment (ranjithwingrider) [reply docs/tsg_install.md]: Done

**Comment by ranjithwingrider** [comment]
Since more update on this page after this PR. so closing this PR and add this entry in the latest page.

**Pull request closed without merging**